l10n_be_account_bank_statement_import_bpost/README.rst

```diff
@@ -0,0 +1,34 @@
+Import Bpost CSV bank statements
```

Comment:
@alexis-via can you just update the README.rst structure according to last template?
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Licence at the top, link to issue tracker and runbot.

I know it's painful to maintain those readmes. At some point they should become partly autogenerated.